client/my-sites/people/invite-status/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from '@automattic/components';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+
+const InviteStatus = ( {
+	type,
+	invite,
+	onResend,
+	handleDelete,
+	resendSuccess,
+	requestingResend,
+	inviteWasDeleted,
+	deletingInvite,
+} ) => {
+	const translate = useTranslate();
+	const { isPending } = invite;
+
+	if ( invite && inviteWasDeleted ) {
+		return null;
+	}
+
+	return (
+		<div
+			className={ classNames( 'people-list-item__invite-status', {
+				'is-pending': isPending,
+				'is-invite-details': type === 'invite-details',
+			} ) }
+		>
+			{ isPending && (
+				<Button
+					className={ classNames( 'people-list-item__invite-resend', {
+						'is-success': resendSuccess,
+					} ) }
+					onClick={ onResend }
+					busy={ requestingResend }
+				>
+					{ resendSuccess ? translate( 'Invite Sent!' ) : translate( 'Resend Invite' ) }
+				</Button>
+			) }
+			<Button
+				className="people-list-item__invite-revoke"
+				busy={ deletingInvite }
```

Comment:
Let's disable this one when `requestingResend` is true. Now you can resend **_and_** revoke an invite.

Reply:
I think `requestingResend` should be added to `deletingInvite`? 🤔

Reply:
I don't think so. Making the button busy implies that the button is doing the action written on it. Disabled makes more sense IMO.

Reply:
Yes, I mistaken busy and disabled, thanks!

Reply:
I agree , disabled makes sense here and will be a nice UX update. I'll quickly add this in this PR.